game_of_life_test_review.py

```diff
@@ -0,0 +1,30 @@
+import pytest
+import game_of_life
+import test
+
+
+def test_newCellStateTrue():
+    assert game_of_life.new_cell_state(game_of_life.first_gen, 1, 3) == "."
+
+
+def test_newCellStateFalse():
+    assert game_of_life.new_cell_state(game_of_life.first_gen, 1, 3) != "x"
+
+
+@pytest.mark.parametrize("listik, x, y, answer", [(game_of_life.first_gen, 1, 3, "."), (game_of_life.first_gen, 0, 3, ".")])
```

Comment:
Nice idea! But why not include the previous two tests here?

Reply:
Those tests are more for training, to be honest. So I've tried to write simple tests and after that I was interested in how to parametrize them (lot's of learning parts here)